sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/DependencyManager.groovy

```diff
@@ -54,22 +78,23 @@ class DependencyManager {
      * Creates a new ProjectOverviewView using
      * <ul>
      *     <li>{@link #projectResourceService}</li>
+     *     <li>{@link #loadProjectsDataSource}</li>
      * </ul>
      * @return a new ProjectOverviewView
      */
-    private createProjectOverviewView() {
-        ProjectOverviewViewModel projectOverviewViewModel = new ProjectOverviewViewModel(projectResourceService)
-        return new ProjectOverviewView(projectOverviewViewModel)
+    private ProjectOverviewView createProjectOverviewView() {
+        ProjectOverviewViewModel viewModel = new ProjectOverviewViewModel(projectResourceService)
+        ProjectOverviewView view =  new ProjectOverviewView(viewModel)
+        return view
     }
 
-    //FIXME remove
-    private void demo() {
-        Project project1 = new Project.Builder(new ProjectIdentifier(new ProjectSpace("My Awesome ProjectSpace 1"), new ProjectCode("QABCD")), "My Awesome Project1").build()
-        Project project2 = new Project.Builder(new ProjectIdentifier(new ProjectSpace("My Awesome ProjectSpace 2"), new ProjectCode("QABCE")), "My Awesome Project2").build()
-        Project project3 = new Project.Builder(new ProjectIdentifier(new ProjectSpace("My Awesome ProjectSpace 3"), new ProjectCode("QABCF")), "My Awesome Project3").build()
-
-        projectResourceService.addToResource(project1)
-        projectResourceService.addToResource(project2)
-        projectResourceService.addToResource(project3)
+    /**
+     * Triggers the project loading initially to have data in the service
+     */
+    void populateProjectService() {
+        LoadProjectsOutput output = new LoadProjectsPresenter(projectResourceService)
+        LoadProjectsInput loadProjects = new LoadProjects(loadProjectsDataSource, output)
+        loadProjects.loadProjects()
     }
```

Comment:
I think this could be moved into the creation method of the respective view? (createProjectOverviewView). 
We did the same in the offer-manager dependency manager.

Reply:
I put it in the dependency manager because it is not called by the view. If we were to put it in the view class the projects would be loaded whenever a view is created. Atm, I think that this is not the desired behaviour and we use one resource service all the time.

Reply:
Ah makes sense. Thanks for clarifying